src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
 public class Application {
+    public static final int NUMBER_OF_LOTTERIES = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Lottery[] lotteries = new Lottery[NUMBER_OF_LOTTERIES];
+
+        for (int i = 0; i < NUMBER_OF_LOTTERIES; i++) {
+            lotteries[i] = lottery;
+            System.out.println(lotteries[i].getRandomBall());
+        }
```

Comment:
Try to understand why it was done that way
```suggestion
        Lottery lottery = new Lottery();

        for (int i = 0; i < NUMBER_OF_LOTTERIES; i++) {
            System.out.println(lottery.getRandomBall());
        }
```